numpy/core/code_generators/genapi.py

```diff
@@ -98,6 +99,33 @@ def _repl(str):
     return str.replace('Bool', 'npy_bool')
 
 
+class MinVersion:
+    def __init__(self, version):
+        """ Version should be the normal NumPy version, e.g. "1.25" """
+        major, minor = version.split(".")
+        self.version = f"NPY_{major}_{minor}_API_VERSION"
+
+    def __str__(self):
+        # Used by version hashing:
+        return self.version
+
+    def add_guard(self, name, normal_define):
+        """Wrap a definition behind a version guard"""
+        numpy_target_help_pointer = (
+            "(Old_NumPy_target_see_depending_on_numpy__"
+            "https://numpy.org/devdocs/dev/depending_on_numpy.html)")
```

Comment:
Maybe its better to just not define it and let people google.  It looked pretty decent when I tried it in godbolt, but when the memory handler tests failed now, it didn't work well (no spaces might help, but I suspect not, since the `:` and `/` will mess things up.

Reply:
seems reasonable to leave this out